build.gradle.kts

```diff
@@ -32,21 +32,21 @@ dependencies {
 
   implementation("com.fasterxml.jackson.datatype:jackson-datatype-jsr310:2.11.2")
 
-  implementation("org.elasticsearch:elasticsearch:7.9.0")
-  implementation("org.elasticsearch.client:elasticsearch-rest-high-level-client:7.9.0")
-  implementation("org.elasticsearch.client:elasticsearch-rest-client:7.9.0")
+  implementation("org.elasticsearch:elasticsearch:7.9.2")
+  implementation("org.elasticsearch.client:elasticsearch-rest-high-level-client:7.9.2")
+  implementation("org.elasticsearch.client:elasticsearch-rest-client:7.9.2")
 
-  implementation("com.amazonaws:aws-java-sdk-core:1.11.854")
+  implementation("com.amazonaws:aws-java-sdk-core:1.11.873")
 
-  implementation("io.springfox:springfox-swagger2:2.9.2")
-  implementation("io.springfox:springfox-swagger-ui:2.9.2")
-  implementation("io.springfox:springfox-bean-validators:2.9.2")
+  implementation("io.springfox:springfox-swagger2:3.0.0")
```

Comment:
Does this work?

Reply:
didn't do when I tried it on other projects

Reply:
would be good to investigate though :grin:

Reply:
I've updated to `io.springfox:springfox-boot-starter:3.0.0` instead after copying @andrewrlee and that worked fine.